lib/media/track/remotevideotrack.js

```diff
@@ -58,7 +59,7 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
         value: new WeakSet(),
       },
       _intersectionObserver: {
-        value: options.IntersectionObserver ? new options.IntersectionObserver(entries => {
+        value: new options.IntersectionObserver(entries => {
```

Comment:
now that `_intersectionObserver` is always non-null, we can remove conditional checks like [this](https://github.com/twilio/twilio-video.js/blob/feature_renderHints/lib/media/track/remotevideotrack.js#L116) and [that](https://github.com/twilio/twilio-video.js/blob/feature_renderHints/lib/media/track/remotevideotrack.js#L96)